### PR TITLE
Setup auto validation of java examples

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

Added auto validation based on manual trigger for now. 

Current Actors example code is not deterministic enough for auto validation.
Also tracing is also not deterministic for validation.

Example successful validation build: https://github.com/mukundansundar/java-sdk/runs/1796635852?check_suite_focus=true

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: Auto Validate Examples
+
+on:
+  workflow_dispatch:
```
Comment (mukundansundar) [inline .github/workflows/validate.yml:9]: For now manual trigger. Can change later.

Comment (wcs1only) [reply .github/workflows/validate.yml:9]: I have it set to run on every push for python-sdk, because why not?

**Review by wcs1only — APPROVED** [review]
Love it!

File: .github/workflows/validate.yml (hunk 1 above)
Comment (artursouza) [reply .github/workflows/validate.yml:9]: Add to pull requests and pushes. This is not different from our ITs.

Comment (artursouza) [reply .github/workflows/validate.yml:9]: Let me fix that.

Comment (artursouza) [reply .github/workflows/validate.yml:9]: Done.

**Review by artursouza — APPROVED** [review]

**Pull request merged**